### PR TITLE
refactor/migrate sdk to prisma

### PR DESCRIPTION
**Pull request by juskek**

### CODE REVIEW

File: sdk-python/omnilogger/database.py
Comment (Lerri-Cofannos) [inline sdk-python/omnilogger/database.py:1]: This is much cleaner, nice!

File: sdk-python/example.env
Comment (Lerri-Cofannos) [inline sdk-python/example.env:1]: If send_to_db does not use the DATABASE_URL anymore then we can remove it from here I believe

File: sdk-python/omnilogger/logger.py [hunk 1]
```diff
@@ -60,13 +61,14 @@ def extract_output_from_response(self, data: str):
...
-        }
-        send_to_db(self.url, log)
+        log = llm_logsCreateInput(
+            input_string=self.prompt,
+            output_string=output["choices"][0]["text"],
+            datetime_utc=datetime.utcfromtimestamp(output["created"]),
+            total_tokens=output["usage"]["total_tokens"],
+        )
+
+        send_to_db(log)
```
Comment (Lerri-Cofannos) [inline sdk-python/omnilogger/logger.py:71]: We can now remove the url argument from the start_listener function I believe

**Review by Lerri-Cofannos — COMMENTED** [review]
If the DATABASE_URL is managed by the prisma.schema then we can remove it in the arguments of the listener

File: prisma/schema.prisma [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 generator client_py {
   provider = "prisma-client-py"
+  interface = "sync"
```
Comment (CallumHemsley) [inline prisma/schema.prisma:5]: what's this do?

Comment (juskek) [reply prisma/schema.prisma:5]: - this generates a synchronous prisma client, 
- the default is an async client, which needs to be awaited 
- the async client was hard to configure with the logger, so I went for the simple solution

File: sdk-python/omnilogger/errors.py
Comment (Lerri-Cofannos) [inline sdk-python/omnilogger/errors.py:1]: This is mind blowing, nice code!

File: sdk-python/omnilogger/input_validation.py [hunk 3]
```diff
@@ -1,27 +1,47 @@
...
+        total_tokens=5,
+    )
+    mockLogWithId = {**log.copy(), "id": -1}
+    mockLogWithId["id"] = -1
+    print(set(log.keys()))
+    print(set(llm_logsCreateInput.__annotations__.keys()))
+    print(
+        set({**log.copy(), "id": -1}.keys())
+        == set(llm_logsCreateInput.__annotations__.keys())
+    )
```
Comment (Lerri-Cofannos) [inline sdk-python/omnilogger/input_validation.py:47]: Is this to do testing? Is it run with pytest?

File: sdk-python/omnilogger/input_validation.py
Comment (Lerri-Cofannos) [inline sdk-python/omnilogger/input_validation.py:1]: This is much cleaner ✨

**Review by Lerri-Cofannos — APPROVED** [review]
Nice improvements!

File: sdk-python/omnilogger/input_validation.py (hunk 3 above)
Comment (juskek) [reply sdk-python/omnilogger/input_validation.py:47]: this should not be here

**Pull request merged**